source/includes/_02-passfort-data-structure.html.md

```diff
@@ -117,8 +117,53 @@ The data provider returned an error message when processing this check.
 This error should be returned when your integration receives a `demo_result`
 value which it doesn't support or does not have demo data implemented.
 
-[api-docs-post-check]:
-https://developer.passfort.com/api#tag/Checks/paths/~1profiles~1{profile_id}~1checks/post
+[api-docs-post-check]: https://developer.passfort.com/api#tag/Checks/paths/~1profiles~1{profile_id}~1checks/post
+[api-docs-openapi-json]: https://identity.passfort.com/api/static/schemas/openapi.json
+
+## Check charges
+
+Checks may return charges in their `charges` field using the structure
+specified here.
 
-[api-docs-openapi-json]:
-https://identity.passfort.com/api/static/schemas/openapi.json
+### Charge fields
+
+<table>
+  <thead>
+    <th>Field</th>
+    <th>Type</th>
+    <th>Required?</th>
+    <th>Description</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>amount</code></td>
+      <td>integer</td>
+      <td>Yes</td>
+      <td>
+        The amount to charge (in pennies). In the future, it may be possible
+        to specify a different currency unit.
+      </td>
+    </tr>
+    <tr>
+      <td><code>sku</code></td>
+      <td>string</td>
+      <td>No</td>
+      <td>
+        Stock Keeping Unit. All charges with the same SKU should cost the
+        same amount. For providers with a fixed cost in each country, the
+        SKU can simply be the 3-letter country code in which the check was
```

Comment:
"...the SKU can simply be the 3-letter code for the country in which the check was run"